opsdroid/connector/matrix/connector.py

```diff
@@ -24,9 +24,11 @@
     Required("mxid"): str,
     Required("password"): str,
     Required("rooms"): dict,
+    Required("store_path"): str,
```

Comment:
let's default this to [`DEFAULT_ROOT_PATH`](https://github.com/opsdroid/opsdroid/blob/6d7a256e88c88c2b7ae9524c5a3c41ebb5848d8a/opsdroid/const.py#L11)`/matrix` for now.